libs/auth/src/dipped_smart_card_auth.ts

```diff
@@ -51,10 +51,8 @@ type AuthAction =
  *
  * See the libs/auth README for notes on error handling
  */
-export class DippedSmartCardAuthWithMemoryCard
-  implements DippedSmartCardAuthApi
-{
-  private authStatus: DippedSmartCardAuth.AuthStatus;
+export class DippedSmartCardAuth implements DippedSmartCardAuthApi {
```

Comment:
Yes good q! There is, mainly for mocking purposes in application-level tests. If we were to have app backends use `DippedSmartCardAuth` directly instead of `DippedSmartCardAuthApi`, we'd have to mock not only the public methods but also all the internal fields and methods of `DippedSmartCardAuth`:

<img width="830" alt="mocking" src="https://user-images.githubusercontent.com/12616928/221042891-38fe5bb5-f76a-4db8-9026-3187ace8a929.png">

Less importantly, there is also something nice about being able to see the public interface at a glance

Reply:
Gotcha, thanks!